src/atomate2/classical_md/openmm/flows/core.py

```diff
@@ -0,0 +1,159 @@
+"""Core flows for OpenMM module."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING
+
+from jobflow import Flow
+
+from atomate2.classical_md.openmm.jobs.core import NVTMaker, TempChangeMaker
+from atomate2.classical_md.utils import create_list_summing_to
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from emmet.core.classical_md import ClassicalMDTaskDocument
+    from openff.interchange import Interchange
+
+    from atomate2.classical_md.openmm.jobs.base import BaseOpenMMMaker
+
+
+@dataclass
+class OpenMMFlowMaker:
```

Comment:
This is an interesting approach to generate a flow but makes sense in the context!